new/components/vcloud/update-vcloud-vm.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+uname -a
+
+# update system
+# yum update --skip-broken -y
+
+# install requirements for python-lxml
+yum install python-devel gcc-c++ gcc libxslt-devel libxml2-devel python-argparse zlib-devel -y
```

Comment:
Well, I guess this is needed for the vCloud api python package itself
right? If so, once we're done with the offline plugins feature, this won't
be needed.
On Oct 16, 2015 12:27 PM, "Denis Pauk" <notifications@github.com> wrote:

> In new/components/vcloud/update-vcloud-vm.sh
> <https://github.com/cloudify-cosmo/cloudify-manager-blueprints/pull/135#discussion_r42224022>
> :
>
> > @@ -0,0 +1,22 @@
> > +#!/bin/bash -e
> > +
> > +uname -a
> > +
> > +# update system
> > +# yum update --skip-broken -y
> > +
> > +# install requirements for python-lxml
> > +yum install python-devel gcc-c++ gcc libxslt-devel libxml2-devel python-argparse zlib-devel -y
>
> Do we can add python-lxml to default cloudify virtual environment that
> used for create deployments? If it can be true we does not need install
> compilers at all.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudify-cosmo/cloudify-manager-blueprints/pull/135/files#r42224022>
> .
>